depmapomics/dm_omics.py

```diff
@@ -725,6 +728,23 @@ def cnPostProcessing(
     merged_feature_table_pr = wgs_feature_table_pr.append(wes_feature_table_pr)
     merged_feature_table_pr.to_csv(folder + "merged_feature_table_profile.csv")
 
+    # merging microsatellite repeats
+    assert wes_ms_df
+    assert wes_ms_df.iloc[:, :5].equals(wgs_ms_df.iloc[:, :5])
```

Comment:
To be 100% sure: `pd.testing.assert_frame_equal(wes_ms_df.loc[:,:5], wgs_ms_df.loc[:,:5])`

Reply:
(just want to make sure we don't end up doing anything screwy with joins and mismatched indexes)

Reply:
Good to know! Thanks!

Reply:
Wait, never mind. I think `equals` used to be inferior to the testing lib method, but reading its docstring it does check the indexes